webapp/packages/core-blocks/src/FormControls/Checkboxes/useCheckboxState.ts

```diff
@@ -56,6 +56,8 @@ export function useCheckboxState<TKey extends string>(options: CheckboxStateOpti
     } else {
       checked = !!currentState;
     }
+  } else if (name !== undefined && state !== undefined) {
+    checked = false;
```

Comment:
looks strange. It should be a part of the first condition imho. like  `if (name in state === false) checked = false`.